dotcom-rendering/src/types/content.ts

```diff
@@ -709,7 +743,8 @@ export type FEElement =
 	| TableBlockElement
 	| TextBlockElement
 	| TimelineAtomBlockElement
-	| TimelineBlockElement
+	| FETimelineBlockElement
+	| DCRTimelineBlockElement
```

Comment:
Can it really be either? Once an `FETimelineBlockElement` is enhanced, it will either result in nothing or a `DCRTimelineBlockElement`…

Reply:
No, it can't. There are several elements in this model that can only exist in DCR, and replace the frontend elements that they're enhanced from. We should really split it into two models: one for frontend data, and one for DCR data once it's been parsed/enhanced.